Source/CesiumRuntime/Private/Tests/CesiumPropertyArray.spec.cpp

```diff
@@ -142,4 +142,51 @@ void FCesiumPropertyArraySpec::Define() {
       }
     });
   });
+
+  Describe("UCesiumPropertyArrayBlueprintLibrary::GetString", [this]() {
```

Comment:
`GetString` is actually deprecated 😅 but it doesn't hurt to leave a test anyways